### PR TITLE
Add ColorPicker Orientation spec

### PR DESCRIPTION
**Pull request by chingucoding**
Add API spec for #1637.

### CODE REVIEW

**Comment by mdtauk** [comment]
Glad to see this proposal coming back to life :)

**Comment by mdtauk** [comment]
Some images you may want to use for Illustration
image
image
----
image
image

**Comment by chingucoding** [comment]
Very good idea @mdtauk, thank you for providing those mockups! Updated the screenshots.

**Comment by robloo** [comment]
As heavily discussed before, redesigning the color picker would be better. The design is deficient in my opinion for this to even be an issue.

#2309

File: specs/ColorPicker/ColorPicker.Orientation.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+```c# (but really MIDL3)
+
+namespace Microsoft.UI.Xaml.Controls
+{
+  [webhosthidden]
+  unsealed runtimeclass ColorPicker
+  {
+      // Existing ...
+
+      // Gets or sets the DisplayMode of the ColorPicker
```
Comment (ptorr-msft) [inline specs/ColorPicker/ColorPicker.Orientation.md:60]: I believe you mean the `Orientation`... :D

Comment (chingucoding) [reply specs/ColorPicker/ColorPicker.Orientation.md]: Indeed haha, thank you @ptorr-msft  for pointing this out! Fixed now.

File: specs/ColorPicker/ColorPicker.Orientation.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+namespace Microsoft.UI.Xaml.Controls
+{
+  [webhosthidden]
+  unsealed runtimeclass ColorPicker
+  {
+      // Existing ...
+
+      // Gets or sets the Orientation of the ColorPicker
+      Windows.UI.Xaml.Controls.Orientation Orientation
```
Comment (oldnewthing) [inline specs/ColorPicker/ColorPicker.Orientation.md:61]:
```suggestion
      Windows.UI.Xaml.Controls.Orientation Orientation;
```

**Review by MikeHillberg — APPROVED** [review]
Thank you @chingucoding!

**Comment by gabbybilka** [comment]
Additions look good to me, thanks @chingucoding!

**Pull request merged**